lib/auth/integration/integrationv1/awsra.go

```diff
@@ -48,14 +52,14 @@ func (s *Service) GenerateAWSRACredentials(ctx context.Context, req *integration
 
 // generateAWSRACredentialsWithoutAuthZ generates a set of AWS credentials which uses the AWS Roles Anywhere integration.
 // Bypasses authz and should only be used by other methods that validate AuthZ.
```

Comment:
We should probably update the godoc to mention the behavior of the new `trustAnchor` argument.